libs/@local/hash-graph-sdk/node/package.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
This needs to work in the browser too so I'd call it `typescript` instead of `node`